browser/app/js/browser/ChangePassword.js

```diff
@@ -99,9 +90,11 @@ export class ChangePasswordModal extends React.Component {
           message: err.message
         })
       })
+
+    this.props.hideChangePassword(this.props.value)
```

Comment:
Why exactly are we passing `this.props.value` here? Looks like it returns `undefined` and the `hideModal` function it is associated with doesn't take any inputs as well.

Reply:
It's unnecessary and look like it's by a mistake. Will fix it.